src/hermes/config.py

```diff
@@ -9,19 +9,63 @@
 import toml
 
 
-_config = {}
+_config = {
+    'logging': {
+        'version': 1,
+
+        'formatters': {
+            'plain': {'format': "%(message)s"},
+            'logfile': {'format': "%(created)16f:%(name)20s:%(levelname)10s | %(message)s"},
+            'auditlog': {'format': "%(asctime)s %(name)-20s  %(message)s"},
+        },
+
+        'handlers': {
+            'terminal': {
+                'class': "logging.StreamHandler",
+                'formatter': "plain",
+                'level': "INFO",
+                'stream': "ext://sys.stdout",
+            },
+
+            'logfile': {
+                'class': "logging.FileHandler",
+                'formatter': "logfile",
+                'level': "DEBUG",
+                'filename': "hermes.log",
+            },
+
+            'auditfile': {
+                'class': "logging.FileHandler",
+                'formatter': "plain",
+                'level': "DEBUG",
+                'filename': "hermes-audit.md",
+                'mode': "w",
+            },
+        },
+
+        'loggers': {
+            'cli': {'level': "DEBUG", 'handlers': ['terminal']},
+            'hermes': {'level': "DEBUG", 'handlers': ['terminal', 'logfile']},
+            'audit': {'level': "DEBUG", 'handlers': ['terminal', 'logfile']},
+        },
+    },
+}
 
 
 def configure():
-    if _config:
+    if 'hermes' in _config:
         return
 
     # Load configuration if not present
-    with open('pyproject.toml', 'r') as config_file:
-        config_toml = toml.load(config_file)
-        hermes_config = config_toml['tool']['hermes']
-        _config['hermes'] = hermes_config
-        _config['logging'] = hermes_config['logging']
+    try:
+        with open('pyproject.toml', 'r') as config_file:
```

Comment:
Does this mean, that if there is a hermes config (maybe from some `hermes.toml`) already loaded, a `pyproject.toml` will be ignored, making it mandatory to have the loggin config in the `hemeres.toml` even if one has a`pyproject.toml`?

Reply:
Yes, there is only one configuration file that should define the behaviour of hermes cli.

Reply:
If it is possible to draw the logging config from `pyproject.toml` if there is no `hernes.toml`, then it would be helpful to be able to do it in either case: maybe someone has the logging config for their project in the `pyproject.toml` and does not want to duplicate the same config in `hermes.toml`. It should also not override an existing config completetly, but only update keys which are set in the new config. THis mechanisn is required anyway to handle defaults.